Resources/assets/js/comments.js

```diff
@@ -353,7 +353,7 @@
                 var reply_button_holder = form.closest('.fos_comment_comment_reply');
 
                 var comment_element = form.closest('.fos_comment_comment_show')
-                    .children('.fos_comment_comment_replies');
+                    .find('.fos_comment_comment_replies:first');
```

Comment:
That's correct. However, have a look at this example:

```
<div class="fos_comment_comment_show">
    <div class="fos_comment_comment_replies">
    </div>
    <div class="fos_comment_comment_replies">
    </div>
</div>
```

In this case `$('.fos_comment_comment_show').children('.fos_comment_comment_replies');` will return two elements while `find('.fos_comment_comment_replies:first');` will only return the first element.